### PR TITLE
Feature/docs update

### PR DESCRIPTION
**Pull request by adamgruber**
- Fixed a few typos
- Reworded/renamed things to be clearer
- Removed unnecessary duplicate variable

### CODE REVIEW

File: src/_variables.scss [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
 /// @type Length
 $gutter-width: 1rem !default;
 
-/// Outer margin to define the space outside the grid
+/// Default grid outer margin
 /// @type Length
 $outer-margin: 2rem !default;
 
-/// Gutter compensation and half with to define the space between columns
-/// @type Length
```
Comment (davegomez) [inline src/_variables.scss:29]: We still need these variables but we have to fix them, they are not supposed to be the same:

```SCSS
$gutter-compensation: (($gutter-width * 0.5) * -1) !default;		
$half-gutter-width: $gutter-width * 0.5 !default;
```

**Review by davegomez — CHANGES_REQUESTED** [review]

File: src/_functions.scss [hunk 2]
```diff
@@ -1,16 +1,15 @@
-/// Calculate the columns and offsets width based in the column count, the
+/// Calculates the columns/offsets width based on the column count, the
```
Comment (jordangehman) [inline src/_functions.scss]: If we're scrutinizing grammar, technically "columns/offsets" is possessive of "width" and should have apostrophes to indicate that. However, I recommend rephrasing it to say "Calculates the width of columns/offsets" instead.

File: src/_functions.scss [hunk 3]
```diff
@@ -12,14 +12,14 @@
   @return ((100 / $columns) * $position) * $unit;
 }
 
-/// Verifies if the value is a percentage or another kind of length in order
-/// to remove or return a percentage value to be used as a multiplier unit
+/// Gets the multiplier unit based on a value.
```
Comment (jordangehman) [inline src/_functions.scss]: Use periods at the end of comment sentences everywhere or nowhere, for consistency.

File: src/_mixins.scss [hunk 4]
```diff
@@ -20,7 +20,7 @@
   #{$property}: get-width($columns, $position, get-multiplier-unit($position));
 }
 
-/// Set the offset width for the provided property based on the column count
+/// Set the offsets width for the provided property based on the column count
```
Comment (jordangehman) [inline src/_mixins.scss]: Same issue with possessiveness. I recommend "Set the width of offsets for…" instead.

File: src/_variables.scss (hunk 1 above)
Comment (jordangehman) [reply src/_variables.scss:29]: @adamgruber updated `_grid.scss` to perform the calculations for gutter adjustments in the property values without referencing these vars. Is that not sufficient?

Comment (davegomez) [reply src/_variables.scss:29]: As long as the calculation is fixed I have no preference for the approach.

Comment (adamgruber) [reply src/_variables.scss:29]: I'll fix the calculation in `_grid.scss`.

**Review by davegomez — APPROVED** [review]

**Pull request merged**